docs/sources/administration/service-accounts/add-service-account-token.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Add a token to a service account in Grafana'
+menuTitle: 'Add a token to a service account'
+description: 'This topic shows you how to add a token to a service account'
+weight: 60
+---
+
+# Add a token to a service account in Grafana
+
+A service account token is a randomly generated string that is used to authenticate with Grafana, with specific permissions to interact with the Grafana HTTP APIs.
+For more information about service accounts, refer to [About service accounts in Grafana]({{< relref "./about-service-accounts.md">}}).
+
+You can create a service account token using the Grafana UI or via the API, for more information about creating a service account token via the API, refer to [HTTP API Create service account token]({{< relref "../../http_api/serviceaccount.md#create-service-account-tokens">}}).
+
+Before you begin:
```

Comment:
```suggestion
## Before you begin
```

Reply:
note of explanation. You are following the style guide correctly, but I have not yet updated that guide with new styling for the before you begin section, which is to format it as an H2 and to not include the colon.

Reply:
il see if i can fix all of the other before you begin